app/src/main/java/com/muzima/view/setupconfiguration/GuidedConfigurationWizardActivity.java

```diff
@@ -46,16 +49,19 @@
 
 import net.minidev.json.JSONObject;
 
+import java.io.IOException;
```

Comment:
@samuelowino is this import necessary?

Reply:
😀 l forgot to remove them, let me resolve that...